python/pyspark/mllib/feature.py

```diff
@@ -174,6 +174,38 @@ def setWithStd(self, withStd):
         self.call("setWithStd", withStd)
         return self
 
+    @property
+    @since('2.0.0')
+    def withStd(self):
+        """
+        Returns if the model scales the data to unit standard deviation.
+        """
+        return self.call("withStd")
+
+    @property
+    @since('2.0.0')
+    def withMean(self):
+        """
+        Returns if the model centers the data before scaling.
+        """
+        return self.call("withMean")
+
+    @property
+    @since('2.0.0')
+    def std(self):
+        """
+        Return the column standard deviation values. Only set if model was trained withStd.
+        """
+        return self.call("std")
+
+    @property
+    @since('2.0.0')
+    def mean(self):
```

Comment:
@holdenk quick question - if `withMean` or `withStd` is `false`, does this return `None` or `0.0`? It is worth adding a little test for this case at all?